oxidation/libparsec/crates/types/tests/test_time.rs

```diff
@@ -1,6 +1,6 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
 
-use crate::{DateTime, Duration};
+use libparsec_types::prelude::*;
```

Comment:
Only for 2 imports ?

Reply:
see https://github.com/Scille/parsec-cloud/pull/4430#discussion_r1171053206